Documentation/Storage-Configuration/NFS/nfs.md

```diff
@@ -0,0 +1,175 @@
+---
+title: NFS Storage Overview
+---
+
+NFS storage can be mounted with read/write permission from multiple pods. NFS storage may be
+especially useful for leveraging an existing Rook cluster to provide NFS storage for legacy
+applications that assume an NFS client connection. Such applications may not have been migrated to
+Kubernetes or might not yet support PVCs. Rook NFS storage can provide access to the same network
+filesystem storage from within the Kubernetes cluster via PVC while simultaneously providing access
+via direct client connection from within or outside of the Kubernetes cluster.
+
+!!! warning
+    Simultaneous access to NFS storage from Pods and from from external clients complicates NFS user
+    ID management significantly.
+
+!!! warning
+    Due to a number of Ceph issues and changes, Rook officially only supports Ceph
+    v16.2.7 or higher for CephNFS. If you are using an earlier version, upgrade your Ceph version
+    following the advice given in Rook's
+    [v1.9 NFS docs](https://rook.github.io/docs/rook/latest/CRDs/ceph-nfs-crd/).
+
+!!! note
+    CephNFSes support NFSv4.1+ access only. Serving earlier protocols inhibits responsiveness after
+    a server restart.
+
+
+## Prerequisites
+
+This guide assumes you have created a Rook cluster as explained in the main
+[quickstart guide](../../Getting-Started/quickstart.md) as well as a
+[Ceph filesystem](../Shared-Filesystem-CephFS/filesystem-storage.md) which will act as the backing
+storage for NFS.
+
+Many samples reference the CephNFS and CephFilesystem example manifests
+[here](https://github.com/rook/rook/blob/master/deploy/examples/nfs.yaml) and
+[here](https://github.com/rook/rook/blob/master/deploy/examples/filesystem.yaml).
+
+
+## Creating an NFS cluster
+
+Create the NFS cluster by specifying the desired settings documented for the
+[NFS CRD](../../CRDs/ceph-nfs-crd.md).
+
+
+## Creating Exports
+
+When a CephNFS is first created, all NFS daemons within the CephNFS cluster will share a
+configuration with no exports defined. When creating an export, it is necessary to specify the
+CephFilesystem which will act as the backing storage for the NFS export.
+
+RADOS Gateways (RGWs), provided by [CephObjectStores](../Object-Storage-RGW/object-storage.md), can
+also be used as backing storage for NFS exports if desired.
+
+### Using the Ceph Dashboard
+
+Exports can be created via the
+[Ceph dashboard](https://docs.ceph.com/en/latest/mgr/dashboard/#nfs-ganesha-management) as well. To
+enable and use the Ceph dashboard in Rook, see [here](../Storage-Configuration/Monitoring/ceph-dashboard.md).
+
+### Using the Ceph CLI
+
+The Ceph CLI can be used from the Rook toolbox pod to create and manage NFS exports. To do so, first
+ensure the necessary Ceph mgr modules are enabled, if necessary, and that the Ceph orchestrator
+backend is set to Rook.
+
+#### **Enable the Ceph orchestrator if necessary**
+
+* Required for Ceph v16.2.7 and below
```

Comment:
Do we still need to document v16.2.7 and older? They could always refer to v1.9 docs if they aren't on rook v1.10, right? Or maybe i'm too ambitious on dropping the old version support.

Reply:
I might leave it. The docs are a copy-paste, and I don't see it hurting anything currently. The section isn't very long.